src/schemas/config.ts

```diff
@@ -9,6 +9,11 @@ export const Config = sqliteTable("guildConfig", {
 
 	gatewayLeaveTitle: text("gatewayLeaveTitle"),
 	gatewayLeaveContent: text("gatewayLeaveContent"),
+
+	suggestionChannel: text("suggestionChannel"),
+	suggestionManagerRole: text("suggestionManagerRole"),
+	suggestionUpvoteEmoji: text("suggestionUpvoteEmoji"),
```

Comment:
I think a default emoji would make sense here rather than in the actual button manager.